rpa-recorder/6.4/modules/ROOT/pages/user-interface.adoc

```diff
@@ -27,6 +28,46 @@ The *Recording Toolbar* appears on the screen when RPA Recorder starts recording
 
 When a recording state changes, a toast notification appears indicating current state. The notification stays for two seconds and disappears automatically.
 
+== Image-Based Mouse Clicks
+
+Use image-based mouse clicks at any point during a normal recording session to record mouse clicks on a UI element that otherwise cannot be recorded properly. This feature guides you through the mouse click process. An image on the screen and mouse clicks must be captured, and the mouse clicks are recorded relative to the captured image.
+
+The feature also checks your display settings automatically while recording to ensure the feature records correctly.
+
+This feature will result in _Search Pattern_ and _Mouse Action_ Action Steps when the workflow is generated in RPA Builder from the recording.
+
+When using this feature:
+
+* The Recording Toolbar is inactive.
+* Keyboard events are not recorded.
+
+=== Recommended Display Settings
+
+Apply the following recommended display settings to capture image-based mouse clicks:
+
+* Single monitor. 
++
+Although the image-based mouse clicks feature supports multiple monitors, that setting limits the capturing to the primary display for Bot tasks and the web page for Web tasks.
```

Comment:
suggestion: correct "that setting limits the capturing to..." to "the capturing is limited to...". The display settings don't limit or restrict the recording (as of now).